README.md

```diff
@@ -49,3 +49,21 @@ Blogging for our community, a blogging website where anyone can write blogs, rea
 7. Style account related pages (AllAuth pages) like forgot password, connect social accounts
 8. **Pitch your own Idea!**
 
+## Text Sentiment Analysis
+
+* *Project link:* https://github.com/khanfarhan10/TextSentimentAnalysis
+* *Mentors:* [Farhan Hai Khan](https://github.com/khanfarhan10) & [Tannistha Pal](https://github.com/paltannistha)
+
+### Description
+
+Text Sentiment Analysis in Python using Natural Language Processing (NLP) for Negative/Positive Content Detection and further classification accordingly to Overwhelming Speech, Appreciation Speech, Demoralising Speech and Hate Speech. Deployed on the Cloud using Streamlit on the Heroku Platform.
+
+### Project Ideas
+
+1. Add models with higher accuracies than the basic and intermediate ones provided with deployment. 
```

Comment:
Please explain a bit more about this, what kind of models you are looking for or which framework you are preferring. What's the current status of the model etc.